Darkplaces-Emscripten/builddeps/buildjpeg.sh

```diff
@@ -0,0 +1,16 @@
+if ! type "emcc" > /dev/null; then
+    printf "\nEmscripten Not installed. Please install Emscripten at https://emscripten.org/docs/getting_started/downloads.html \n"
+    exit 1
+fi
+#doesn't work on codespaces but works on a normal computer
+git clone https://github.com/libjpeg-turbo/libjpeg-turbo .jpeg
```

Comment:
`libjpeg-dev` is already a requirement. Emscriptem/WASM specifically requires `libjpeg-turbo`?

Reply:
Won’t work. The library’s .a file needs to be compiled to WASM. Also, emscripten’s port of libjpeg just doesn’t work.